airflow/dags/maticetl_airflow/build_export_dag.py

```diff
@@ -234,10 +235,20 @@ def export_traces_command(execution_date, provider_uri, **kwargs):
                 start_block=start_block,
                 end_block=end_block,
                 batch_size=export_batch_size,
-                output=os.path.join(tempdir, "traces.csv"),
+                output=os.path.join(tempdir, "geth_traces.json"),
                 max_workers=export_max_workers,
                 provider_uri=provider_uri
             )
+            copy_to_export_path(
```

Comment:
@araa47 I added this so that raw traces are uploaded to GCS, in case we need to reprocess them later